velox/connectors/hive/storage_adapters/s3fs/S3Util.h

```diff
@@ -57,18 +65,32 @@ inline Aws::String awsString(const std::string& s) {
   return Aws::String(s.begin(), s.end());
 }
 
-#define VELOX_CHECK_AWS_OUTCOME(outcome, header, bucket, key) \
-  {                                                           \
-    if (!outcome.IsSuccess()) {                               \
-      auto error = outcome.GetError();                        \
-      VELOX_FAIL(                                             \
-          "{} with bucket '{}' and key '{}' due to {}:{}",    \
-          header,                                             \
-          bucket,                                             \
-          key,                                                \
-          error.GetErrorType(),                               \
-          error.GetMessage());                                \
-    }                                                         \
+std::string getErrorStringFromS3Error(
```

Comment:
It would be nice to document this function.

Reply:
I will do it as part of https://github.com/facebookincubator/velox/issues/864